### ISSUE TITLE
Build out testing

### ISSUE THREAD

**Issue by pjsier** [national-voter-file/national-voter-file#203]
# Problem

We only have one test per state currently that just runs the transformer on fake data for that state. We should make sure we're covering more scenarios including preparer objects accepting zip files instead of text files (for states where that's the case) as well as testing processing voter history when that gets set up.

**Comment by joeymbryan** [comment]
Hey @pjsier I'm new to Prog Coders and new to testing but would love to help. Perhaps we could setup a meeting to go over the project and what testing is already in place?

**Comment by pjsier** [comment]
@joeymbryan Definitely! Right now all of our tests are [here](https://github.com/national-voter-file/national-voter-file/blob/master/src/python/national_voter_file/tests/test_transformers.py), and really just check whether running any voter file breaks the whole thing. I can send you a message on the ProgCode Slack to set up some time to talk through it more if that's helpful
